### PR TITLE
scripts:add description for debian package

### LINKED ISSUE #1718: Add description for apt package
As described by user:
https://discordapp.com/channels/485586884165107732/485598848111083531/555327757475905546

We should write some description for our apt package.

### PR DESCRIPTION
**Pull request by kurianbenoy**
Fix #1718 
* [x] Have you followed the guidelines in our
      [Contributing document](https://dvc.org/doc/user-guide/contributing)?

* [x] Does your PR affect documented changes or does it add new functionality
      that should be documented? If yes, have you created a PR for
      [dvc.org](https://github.com/iterative/dvc.org) documenting it or at
      least opened an issue for it? If so, please add a link to it.

-----

### CODE REVIEW

**Comment by efiop** [comment]
@kurianbenoy did you test that?

File: scripts/build_posix.sh [hunk 1]
```diff
@@ -46,6 +46,7 @@ fpm_build()
 	fpm -s dir \
 	    -f \
 	    -t $1 \
+		--description "Data Version Control - datasets, models, and experiments versioning for ML or data science projects"
```
Comment (efiop) [inline scripts/build_posix.sh]: let's define DESC along others (see the top of this file) and use it here something like `--description 
$DESC`.

Comment (efiop) [reply scripts/build_posix.sh]: Also, you've forgot `\` at the end of the line.

**Comment by kurianbenoy** [comment]
> @kurianbenoy did you test that? slightly_smiling_face

How can I test it @efiop, is it by running the script file? Black8 tests passed for this

**Comment by efiop** [comment]
@kurianbenoy well, black/flake8 are testing for code style. You could test that script by running it from the root of the repo, yes.

**Comment by shcheklein** [comment]
@kurianbenoy any updates on this? do you need any help with this? :)

**Comment by kurianbenoy** [comment]
@efiop I have tested the script and it works. I have made the requested changes you mentioned

File: scripts/build_posix.sh [hunk 2]
```diff
@@ -15,6 +15,7 @@ fi
 
 BUILD_DIR=build
 BIN_DIR=$BUILD_DIR/$INSTALL_DIR/bin
+DESC='Data Version Control - datasets, models, and experime    nts versioning for ML or data science projects'
```
Comment (efiop) [inline scripts/build_posix.sh]:
```suggestion
DESC='Data Version Control - datasets, models, and experiments versioning for ML or data science projects'
```

File: scripts/build_posix.sh [hunk 3]
```diff
@@ -46,6 +47,7 @@ fpm_build()
 	fpm -s dir \
 	    -f \
 	    -t $1 \
+		--description $DESC \
```
Comment (efiop) [inline scripts/build_posix.sh]:
```suggestion
	    --description $DESC \
```

**Comment by efiop** [comment]
Thanks @kurianbenoy . Please see a few more comments.

File: scripts/build_posix.sh (hunk 3 above)
Comment (kurianbenoy) [reply scripts/build_posix.sh]: Should I remove two spaces or add two spaces

**Comment by kurianbenoy** [comment]
@efiop whenever I am committing into dvc, is it commulsory to have` pre-commit and black installed`?
I am using a Debian system with python3.5 so everytime, I need to run in docker and again install dependencies, thats why I asked

**Comment by efiop** [comment]
> @efiop whenever I am committing into dvc, is it commulsory to have pre-commit and black installed?I am using a Debian system with python3.5 so everytime, I need to run in docker and again install dependencies, thats why I asked

Yes, for development it is compulsory to have pre-commit and black, so that they check your commits before they are submitted. Maybe take a look at `pyenv`? I'm using it myself to switch between python versions.

**Comment by kurianbenoy** [comment]
> > @efiop whenever I am committing into dvc, is it commulsory to have pre-commit and black installed?I am using a Debian system with python3.5 so everytime, I need to run in docker and again install dependencies, thats why I asked
> 
> Yes, for development it is compulsory to have pre-commit and black, so that they check your commits before they are submitted. Maybe take a look at `pyenv`? I'm using it myself to switch between python versions.

Thanks for the suggestion, will check it out

**Pull request merged**